### ISSUE TITLE
pg-config-executable-not-found - linux

### ISSUE THREAD

**Issue by sethbergman** [OperationCode/operationcode#619]
As an ADMIN:
When I try to build the application with `bundle`, the build fails to locate `pg_config`, even though it's there and already added to my $PATH. 

It should recommend installing the appropriate libraries, but it recommends installing postgresql-server-dev-X.Y, which is not an actual package. 

I did some research, and found a [StackOverflow article](http://stackoverflow.com/questions/11618898/pg-config-executable-not-found) that had an answer that fit this specific situation - based on the order of the setup instructions. What ended up working for me is adding the following packages:
`sudo apt-get install libpq-dev python-dev`

![pg-bug](https://cloud.githubusercontent.com/assets/8031689/22773918/45319e4a-ee6a-11e6-9acb-47868839dac8.png)

Additional Information:
```
cat /proc/version
Linux version 4.4.0-62-generic (buildd@lcy01-30) (gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.4) ) #83-Ubuntu SMP Wed Jan 18 14:10:15 UTC 2017
```

#### Adding this to CONTRUBUTING.md should resolve this issue. :rocket:

**Comment by kboygit** [comment]
@sethbergman spot on , thank you

**Comment by rickr** [comment]
Fixed in #626
